spec/get_visits_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'get all visits route', type: :request do
+  let!(:visits) { FactoryBot.create_list(:random_visit, 20) }
+
+  before { get '/api/v1/visits' }
```

Comment:
Lets use generators like 'rails g rspec:model <model_name>' for building spec structure